src/sentry/static/sentry/app/views/onboarding/wizardNew.jsx

```diff
@@ -0,0 +1,289 @@
+import {browserHistory} from 'react-router';
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import React from 'react';
+import posed, {PoseGroup} from 'react-pose';
+import scrollToElemennt from 'scroll-to-element';
+import styled from 'react-emotion';
+
+import {analytics} from 'app/utils/analytics';
+import {t} from 'app/locale';
+import InlineSvg from 'app/components/inlineSvg';
+import OnboardingPlatform from 'app/views/onboarding/platform';
+import OnboardingProjectSetup from 'app/views/onboarding/projectSetup';
+import OnboardingWelcome from 'app/views/onboarding/welcome';
+import PageHeading from 'app/components/pageHeading';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import withOrganization from 'app/utils/withOrganization';
+import withProjects from 'app/utils/withProjects';
+
+const recordAnalyticStepComplete = ({organization, project, step}) =>
+  analytics('onboarding_v2.step_compete', {
+    org_id: parseInt(organization.id, 10),
+    project: project ? project.slug : null,
+    step: step.id,
+  });
+
+const ONBOARDING_STEPS = [
+  {
+    id: 'welcome',
+    title: t('Welcome to Sentry'),
+    Component: OnboardingWelcome,
+  },
+  {
+    id: 'select-platform',
+    title: t('Select a platform'),
+    Component: OnboardingPlatform,
+  },
+  {
+    id: 'get-started',
+    title: t('Install the Sentry SDK'),
+    Component: OnboardingProjectSetup,
+  },
+];
+
+class OnboardingWizard extends React.Component {
+  static propTypes = {
+    projects: PropTypes.arrayOf(SentryTypes.Project),
+    organization: SentryTypes.Organization,
+  };
+
+  state = {};
+
+  get activeStepIndex() {
+    return ONBOARDING_STEPS.findIndex(({id}) => this.props.params.step === id);
+  }
+
+  get activeStep() {
+    return ONBOARDING_STEPS[this.activeStepIndex];
+  }
+
+  get firstProject() {
+    const sortedProjects = this.props.projects.sort(
+      (a, b) => new Date(b.dateCreated) - new Date(a.dateCreated)
+    );
+
+    return sortedProjects.length > 0 ? sortedProjects[0] : null;
+  }
+
+  get projectPlatform() {
+    return this.state.platform || this.firstProject?.platform;
+  }
+
+  handleUpdate = data => {
+    this.setState(data);
+  };
+
+  handleNextStep(step, data) {
+    this.handleUpdate(data);
+
+    if (step !== this.activeStep) {
+      return;
+    }
+
+    const {orgId} = this.props.params;
+    const nextStep = ONBOARDING_STEPS[this.activeStepIndex + 1];
```

Comment:
Do you need to handle advancing past the last step?

Reply:
Currently no.